env.yml

```diff
@@ -0,0 +1,165 @@
+name: coffea
```

Comment:
Regenerate with `--from-history` to remove most of the unneeded versioning

Reply:
done